config/betterquesting/DefaultQuests.json

```diff
@@ -4473,7 +4473,9 @@
     "47:10": {
       "questID:3": 47,
       "preRequisites:11": [
-        46
+        46,
+        76,
+        948
```

Comment:
I am not ingame what quests are 76 and 948

Reply:
or add more text in your pr text field. Thanks

Reply:
76 - Time to Get Some Steel
948 - Better Than Iron
46 - More Advanced Alloys

Reply:
Thanks @bombcar